tests/functional/testOnboarding.js

```diff
@@ -205,13 +203,17 @@ describe('Onboarding', function() {
     assert.equal(await vpn.getQueryProperty(queries.screenSettings.privacyView.BLOCK_ADS_CHECKBOX, 'checked'), 'true');
     assert.equal(await vpn.getQueryProperty(queries.screenSettings.privacyView.BLOCK_TRACKERS_CHECKBOX, 'checked'), 'true');
     assert.equal(await vpn.getQueryProperty(queries.screenSettings.privacyView.BLOCK_MALWARE_CHECKBOX, 'checked'), 'true');
+    assert.equal(await vpn.getSetting('dnsProviderFlags'), 14);
```

Comment:
What is 14 here? I see DNS Provider flags defined as following in the code. They would be 0, 1, 2, 4, 8 🤔 

```c++
  enum DNSProviderFlags {
    Gateway = 0x00,
    Custom = 0x01,
    BlockAds = 0x02,
    BlockTrackers = 0x04,
    BlockMalware = 0x08,
  };
```

Reply:
Why is this part of this PR?

Reply:
Can you add a comment to explain the math/logic for that line?